include/irqmgr.h

```diff
@@ -3,6 +3,17 @@
 
 #include "ultra64.h"
 
+#define OS_SC_RETRACE_MSG       1
+#define OS_SC_DONE_MSG          2
+#define OS_SC_NMI_MSG           3 // name is made up, 3 is OS_SC_RDP_DONE_MSG in the original sched.c
```

Comment:
This refers to the libultra sched module, which was source-available so as I understand it devs would often make changes. In this case the deviation is appropriate since the event this message is associated with isn't related to the RDP